source/sass/buyers-guide/bg-main.scss

```diff
@@ -21,6 +21,7 @@ $bp-xl: #{map-get($grid-breakpoints, xl)}; // >= 1200px
 //Site-wide
 
 @import '../colors';
+@import './colors'; // Buyer's Guide specific colors
```

Comment:
Buyer's Guide design doesn't align 100% with the main site... colour-wise especially. We could drop the variable in the main `colors.scss` but since now the main site doesn't use that colour, I created a separate Buyer's Guide `color.scss` for it.

Reply:
We will be adding more variables to that color.scss as we go through Buyer's Guide SCSS code to clean up colour codes.